src/Common/MorganStanley.ComposeUI.Common.Utilities/MorganStanley.ComposeUI.Common.Utilities.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<!--
+********************************************************************************************************
+
+Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+
+********************************************************************************************************
+-->
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NP.Concepts.Behaviors" Version="1.0.9" />
```

Comment:
What's this package?

Reply:
+1 on this. Can we include less dependencies on packages maintained by a single person?

Reply:
Avalonia needs some third party tooling, including UniDock. UniDock depends on this package as well as NP.Avalonia.Visuals and NP.Utilities. This is all open source with plenty of samples. There are no other Docking Avalonia packages that do the job, so we are kind of forced to use it.

Reply:
I still have to keep a refernce to NP.Utilities, until we resolve our disagreements about an IoC framework. I'll remove this as part of IoC Jira https://github.com/morganstanley/ComposeUI/issues/68.